homeassistant/components/portainer/__init__.py

```diff
@@ -57,4 +57,10 @@ async def async_migrate_entry(hass: HomeAssistant, entry: PortainerConfigEntry)
         data[CONF_API_TOKEN] = data.pop(CONF_API_KEY)
         hass.config_entries.async_update_entry(entry=entry, data=data, version=2)
 
+    if entry.version < 3:
+        data = dict(entry.data)
+        # Verify SSL was never set, so at first set it to False
+        data[CONF_VERIFY_SSL] = False
```

Comment:
```suggestion
        data[CONF_VERIFY_SSL] = True
```

https://github.com/home-assistant/core/pull/152955 adds this attribute, and before the certificate was always verified. For that reason, we should set it to `True`